SQL_module/SubjectForm/index.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+    function func(String $text, int $size, &$valid, &$errors){
```

Comment:
Maybe func is not the best name for this functionality. You can replace it by `errorHandler()` or something similar.